src/renderer/routes/test_sequencer_panel/components/TestSequencerInfo.tsx

```diff
@@ -98,14 +105,14 @@ const TestSequencerView = () => {
                   >
                     Save Test Set
                   </LockableButton>
-                  <LockableButton
+                  <Button
                     variant="dotted"
                     className="mt-4 w-full gap-2"
-                    isLocked={_.isEmpty(tree)}
-                    onClick={handleClickRunTest}
+                    disabled={backendState === "TEST_SET_EXPORT"}
```

Comment:
Quick question: have you considered using the lockable button? During a specific state of the test sequencer, you can chose to have them locked or not without needing to specify the disabled. The thing is, once locked, all the buttons defined as Lockable will also be locked. Just food for thought tho.

Reply:
It's because I need to Unlock this button even when other are lock. But there is a case where I want to keep it lock (when we are uploading). I'll double check how I can do it